README.md

```diff
@@ -42,7 +42,7 @@ Following JARs are needed from `<KAFKA_HOME>/libs` directory.
 
 ## Installation
 
-For installing this extension and to add the dependent jars on various siddhi execution environments refer Siddhi documentation section on <a target="_blank" href="https://siddhi.io/redirect/add-extensions.html">adding extensions and jars</a>.
+For installing this extension and to add the dependent jars on various siddhi execution environments refer Siddhi documentation section on <a target="_blank" href="https://siddhi.io/en/v5.1/docs/guides/fault-tolerance/guide/#setup-kafka">setup kafka</a>.
```

Comment:
Don't reference the guides. Refer the documentation in https://siddhi.io/en/v5.1/docs/config-guide/#adding-extensions-and-third-party-dependencies  and add other needed information here itself.

Reply:
Fixed in https://github.com/siddhi-io/siddhi-io-kafka/pull/119/commits/9bf1603b6347b96af325299aa98d7d77803180a7